lib/block-supports/elements.php

```diff
@@ -200,11 +189,43 @@ function gutenberg_render_elements_support_styles( $pre_render, $block ) {
 		}
 	}
 
-	return null;
+	return $parsed_block;
+}
+
+/**
+ * Ensure the elements block support class name generated and added to
```

Comment:
This seems to be a bug; I'd expect it to not be necessary to add the attributes manually to `$block_content`. It might be [this issue](https://core.trac.wordpress.org/ticket/56417) resurfacing. I was meaning to fix it at the time and then forgot all about it 😅

Reply:
My inclination was that it was due to the block being static and the markup having been generated at the point it was saved rather than render.